From a3a4eb2fef8324421298f29a954341369b3dddbb Mon Sep 17 00:00:00 2001
From: Roland Stigge <stigge@antcom.de>
Date: Sun, 19 Aug 2012 13:04:29 +0200
Subject: [PATCH 555/611] usb: phy: isp1301: Remove unused static array and
 define

This patch removes an unused statically defined array and an associated
#define.

Signed-off-by: Roland Stigge <stigge@antcom.de>
Signed-off-by: Felipe Balbi <balbi@ti.com>
(cherry picked from commit e7aafd72f3bbf5032db5e96dd3b130e0077513ee)
---
 drivers/usb/phy/isp1301.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/usb/phy/isp1301.c b/drivers/usb/phy/isp1301.c
index b19f493..18dbf7e 100644
--- a/drivers/usb/phy/isp1301.c
+++ b/drivers/usb/phy/isp1301.c
@@ -15,12 +15,6 @@
 
 #define DRV_NAME		"isp1301"
 
-#define ISP1301_I2C_ADDR	0x2C
-
-static const unsigned short normal_i2c[] = {
-	ISP1301_I2C_ADDR, ISP1301_I2C_ADDR + 1, I2C_CLIENT_END
-};
-
 static const struct i2c_device_id isp1301_id[] = {
 	{ "isp1301", 0 },
 	{ }
-- 
1.8.2

